server/utils/addNewRowToGoogleSheets.ts

```diff
@@ -25,13 +26,21 @@ export const addNewRowToGoogleSheets = async (SSAEvents: SSAEvent[]) => {
       return;
     }
     for (const row of SSAEvents) {
-      const { identityId, displayName, event } = row;
+      const { identityId, displayName, event, urlSlug } = row;
 
       const now = new Date();
       const formattedDate = now.toISOString().split("T")[0];
       const formattedTime = now.toISOString().split("T")[1].split(".")[0];
 
-      const dataRowToBeInsertedInGoogleSheets = [formattedDate, formattedTime, identityId, displayName, "Quest", event];
+      const dataRowToBeInsertedInGoogleSheets = [
+        formattedDate,
+        formattedTime,
+        identityId,
+        displayName,
+        "Quest",
+        event,
+        urlSlug,
+      ];
 
       // @ts-ignore
       await sheetsClient.spreadsheets.values.append({
```

Comment:
I think it's slightly better to send this request in one go instead of one by one in a for loop. You can store the data to be sent in a 2D array and send it once. I did that for [chess](https://github.com/metaversecloud-com/sdk-chess-game/blob/main/server/utils/addNewRowToGoogleSheets.ts). It also has the added benefit of putting the same timestamp on events that happen at the same time.